### DISCUSSION TITLE
请问大陆白名单模式有两个DNS可选，是什么意思

### DISCUSSION THREAD

**Discussion by CATinTIN** [immortalwrt/homeproxy#135] [Q&A]
GFW模式就只有一个DNS设置。大陆白名单模式不应该设置国内DNS，然后解析境外域名DNS交给代理跳板吗？
我选择了1.1.1.1，然后比较卡，不得已只好用GFW模式

**Comment by 1715173329** [comment]
不知道可以保持默认。
